constants/constants.go

```diff
@@ -1,3 +1,5 @@
+// Package constants provide list of constant values
+// for the names of form elements and file type.
```

Comment:
raise an issue for adding style guides to the project to make sure that any piece of code conforms to the agreed-upon coding style and best practices.